test/00-init.spec.ts

```diff
@@ -33,4 +34,3 @@ fs.writeFileSync(
 import './lib/mocked-dbus';
 import './lib/mocked-dockerode';
 import './lib/mocked-iptables';
-import './lib/mocked-event-tracker';
```

Comment:
This file only contains a `sinon.spy` call to the event-tracker module. We should make sinon calls when needed within individual suites - having a global sinon stub/spy is not intuitive since you need to remember to import that stub/spy in tests when you need it, or risk a `Attempted to wrap ${FN} which is already wrapped` error.